tests/src/Template/Provider/VcsProviderTest.php

```diff
@@ -3,13 +3,13 @@
 declare(strict_types=1);
 
 /*
- * This file is part of the TYPO3 CMS extension "cpsit/project-builder".
+ * This file is part of the Composer package "cpsit/project-builder".
```

Comment:
Darn it. That's the result of a very frequently used snippet in my IDE. Apologies!